### PR TITLE
Adjust allowed content types

### PR DESCRIPTION
**Pull request by atanasdinov**
# Description

## What

`notification-creator` will publish notifications for internal content types where applicable so the allow-listed ones should be adjusted.

## Why

[JIRA Ticket](https://financialtimes.atlassian.net/browse/UPPSF-2470)

## Scope and particulars of this PR (Please tick all that apply)

- [ ] Tech hygiene (dependency updating & other tech debt)
- [ ] Bug fix
- [x] Feature
- [ ] Documentation
- [ ] Breaking change
- [ ] Minor change (e.g. fixing a typo, adding config)

___
This Pull Request follows the rules described in our [Pull Requests Guide](https://github.com/Financial-Times/upp-docs/tree/master/guides/pr-guide)

### CODE REVIEW

File: consumer/content.go [hunk 1]
```diff
@@ -96,7 +96,7 @@ func (qHandler *ContentQueueHandler) HandleMessage(queueMsg kafka.FTMessage) err
 	}
 	notification.IsE2ETest = isE2ETest
 
-	qHandler.log.WithField("resource", notification.APIURL).WithField("transaction_id", notification.PublishReference).Info("Valid notification received")
+	monitoringLogger.WithField("resource", notification.APIURL).WithField("transaction_id", notification.PublishReference).Info("Valid notification received")
```
Comment (atanasdinov) [inline consumer/content.go:99]: I'd like to go to `staging` with the extended logging and remove it before going to `prod`.

Comment (StephanZahariev) [reply consumer/content.go:99]: Why remove it on PROD? Some additional fields wont hurt PROD IMHO.

Comment (atanasdinov) [reply consumer/content.go:99]: I'm also fine with keeping those extra fields (`contentType` and `event`) here. Would definitely make debugging it easier.

File: helm/notifications-push/app-configs/notifications-push_eks_delivery.yaml [hunk 2]
```diff
@@ -8,7 +8,7 @@ env:
   CONSUMER_BACKOFF: "2"
   NOTIFICATIONS_RESOURCE: "content"
   CONTENT_URI_WHITELIST: "^http://(methode|wordpress-article|content|upp)(-collection|-content-placeholder|-notifications-creator)?(-mapper|-unfolder|)(-pr|-iw)?(-uk-.*)?\\.svc\\.ft\\.com(:\\d{2,5})?/(content|complementarycontent)/[\\w-]+.*$"
-  CONTENT_TYPE_WHITELIST: "application/vnd.ft-upp-article+json,application/vnd.ft-upp-content-package+json,application/vnd.ft-upp-audio+json,application/vnd.ft-upp-live-blog-post+json,application/vnd.ft-upp-live-blog-package+json"
+  CONTENT_TYPE_WHITELIST: "application/vnd.ft-upp-article-internal+json,application/vnd.ft-upp-content-package+json,application/vnd.ft-upp-audio+json,application/vnd.ft-upp-live-blog-post-internal+json,application/vnd.ft-upp-live-blog-package-internal+json"
```
[the remark this answers is not in the document]
Comment (atanasdinov) [reply helm/notifications-push/app-configs/notifications-push_eks_delivery.yaml:11]: All non-internal types are filtered out by `upp-notifications-creator` - they won't ever make it to the topic `notifications-push` is reading from. The reason behind this is that we will deduce the proper notification type (`UPDATE` or `CREATE`) from an internal field.

**Review by StephanZahariev — APPROVED** [review]

**Review by Zarrie — APPROVED** [review]

**Review by gotha — APPROVED** [review]

**Pull request merged**